.github/workflows/ci.yml

```diff
@@ -145,6 +145,8 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install torch
         run: pip install --timeout 300 torch==1.13.1
+      - name: Pip freeze
```

Comment:
why is this needed?

Reply:
Debug, it looks like the CI is failing because of new PyG version, this will hopefully get resolved with Ray fix so I will finalize then.